canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -134,6 +154,16 @@ export default class PropertiesController {
 		return this.propertiesConfig.containerType === "Tearsheet";
 	}
 
+	// Compare old & new controls, set an array of different properties
+	setDifferentProperties() {
+		const differentProperties = PropertyUtils.getObjectDifference(this.controls, this.oldControls);
```

Comment:
Would lodash.differenceBy do this same thing?  I think we just need to compare the keys of the objects.  Right?  We only need to compute the once so I don't know if we need to store differences as a class variable.

Reply:
Tried using `differenceBy(this.controls, this.prevControls, isEqual)` but its returning an empty array.

Reply:
You'd probably write your own function to compare the control names.

Reply:
or maybe just pass in the keys since that's really what you to know the difference on.  Then I think you can use just difference